service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisTemplateDataGeneration.java

```diff
@@ -20,11 +20,24 @@ public static String generateDraftWatermarkEncodedString() throws IOException {
         return Base64.getEncoder().encodeToString(fileContent);
     }
 
+    public static String generateCourtSealEncodedString() throws IOException {
+        InputStream is = DocmosisTemplateDataGeneration.class
+            .getResourceAsStream("/assets/images/family-court-seal.png");
```

Comment:
maybe a q for @akomisarek... is it fine to store a seal in a publicly available repo?

Reply:
I'm checking with Adam on this one

Reply:
Adam and Jay fine with this - it's available on the internet so is open